docs/index.rst

```diff
@@ -1,28 +1,74 @@
-.. _index:
+.. _page-title:
```

Comment:
since the name of the page changed, I'm getting a build error because of the subpage.rst
```
Warning, treated as error:
/home/ubuntu/Documents/GitHub/crate-docs-theme/docs/subpage.rst:9:undefined label: index
```
Changing the <index> to <page-title> in that line solved the issue, not sure if that's what it's supposed to be ;)